src/diffusers/models/autoencoders/autoencoder_kl_cogvideox.py

```diff
@@ -993,8 +1048,32 @@ def encode(
             return (posterior,)
         return AutoencoderKLOutput(latent_dist=posterior)
 
+    def _decode(self, z: torch.Tensor, return_dict: bool = True) -> Union[DecoderOutput, torch.Tensor]:
+        if self.use_tiling and (z.shape[-1] > self.tile_latent_min_width or z.shape[-2] > self.tile_latent_min_height):
+            return self.tiled_decode(z, return_dict=return_dict)
+
+        frame_batch_size = self.num_latent_frames_batch_size
+        dec = []
+        for i in range(z.shape[2] // frame_batch_size):
```

Comment:
Small nit to improve clarity. I would add batch_size, channels, frames, height, width = z.shape and change these lines to iterate over that.